python/ray/autoscaler/_private/fake_multi_node/docker_monitor.py

```diff
@@ -110,7 +110,7 @@ def _update_docker_status(
     data_str = ""
     try:
         data_str = (
-            subprocess.check_output(
+            subprocess.check_call(
```

Comment:
Probably not this one. This call requires `data_str` return value from `check_output`

Reply:
true..